docs/src/api/csharp.md

```diff
@@ -1,10 +1,13 @@
-## method: Request.getPayloadAsJson
+## method: Request.PostDataJsonAsync
```

Comment:
All we need is a return type override, right?

Call it as the origianal, "## method: Request.postDataJSON" so that we could convert it to the override in a follow-up.